src/OpenTelemetry.Exporter.OpenTelemetryProtocol/ExportProtocol.cs

```diff
@@ -0,0 +1,18 @@
+namespace OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// Supported by OTLP exporter protocol types according to the specification https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md.
+    /// </summary>
+    public static class ExportProtocol
```

Comment:
Should we make this an enum?

Reply:
I was considering enum as well, but in context of it's usage it doesn't bring anything with exception of additional mapping of configuration values to enum members. These constants are defined in spec.

Reply:
I think what enum brings in this case would be the ease of use for developers. Consider I do this...

```csharp
            var options = new OtlpExporterOptions
            {
                Protocol = | <- cursor is here
```

At the moment `Protocol` is a string so I don't get any intellisense help. No way to know `ExportProtocol` class exists with constants. If it was an enum, IDE is going to show me immediately the available choices. Also prevents people from doing invalid things like...

```csharp
            var options = new OtlpExporterOptions
            {
                Protocol = "UDP"
            };       
```

Reply:
Transformed to enum.

Reply:
Note, that after introducing enum instead of string setting of Protocol option in code in `OtlpExporterOptions` and using environment variables will look a bit different. Environment variable values should follow [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md). I hope it will not confuse developers :-)